### PR TITLE
Use animation frame uint32_t instead of timestamp float

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: brayns/common/vocabulary/parameters.fbs [hunk 1]
```diff
@@ -146,7 +146,7 @@ enum Engine: string {
 
 table Settings {
     engine: Engine;
-    timestamp: float;
+    animation_frame: uint;
```
Comment (favreau) [inline brayns/common/vocabulary/parameters.fbs]: I think this one should be removed. It duplicates the frame event.

Comment (tribal-tec) [reply brayns/common/vocabulary/parameters.fbs]: Good point. Done

**Comment by hernando** [comment]
Just curious. Why changing this to use frame numbers when the compartment report reader uses time stamps and spike datasets have no frames at all? Additionally, I really think that frame numbers is not what users should be seeing, is this change becoming visible in the public API?

**Comment by tribal-tec** [comment]
The animation frame is used for other things as well, like volume time series, morphology growing etc. It was only used so far as a frame, like increasing/decreasing by 1 only, no fractions involved. The only place where it actually needs timestamps is the compartment report. No spikes report in Brayns. The API already exposed the frame, the timestamp was redundant and wrong as it represented the frame number as well.

**Pull request merged**